<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="editSearch"/>
      </item>
      <item>
       <widget class="QPushButton" name="buttonFind">
        <property name="text">
         <string>Найти</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTextEdit" name="textBrowser"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="putForm" stretch="0,0,0,1,0,0,0,0,0">
      <item>
       <widget class="QLabel" name="idLabel">
        <property name="text">
         <string>ID</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="putID"/>
      </item>
      <item>
       <widget class="QLabel" name="nameLabel">
        <property name="text">
         <string>Full Name</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="putName"/>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>YOB</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="putYear"/>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Gender</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="genderBox">
        <item>
         <property name="text">
          <string>Male</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Female</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="putBut">
        <property name="text">
         <string>Добавить</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="saveBut">
      <property name="text">
       <string>С О Х Р А Н И Т Ь</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>putBut</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>addEmployee()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>767</x>
     <y>523</y>
    </hint>
    <hint type="destinationlabel">
     <x>625</x>
     <y>511</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonFind</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>findEmployee()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>740</x>
     <y>38</y>
    </hint>
    <hint type="destinationlabel">
     <x>639</x>
     <y>59</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>saveBut</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>saveFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>403</x>
     <y>555</y>
    </hint>
    <hint type="destinationlabel">
     <x>189</x>
     <y>510</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>findEmployee()</slot>
  <slot>addEmployee()</slot>
  <slot>saveFile()</slot>
 </slots>
</ui>
